src/components/Error404.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactElement } from 'react';
+import SVGFile from '../errorIcon.svg';
+
+const Error404 = ():ReactElement => {
+    return (
+        <div>
+            <div className="flex flex-col items-center justify-center mx-2xl min-h-[70vh]">
```

Comment:
what's mx-2xl? I might be tripping just couldn't find it in the tailwind docs

Reply:
Ally I think I was tripping when I wrote it, it seems to be doing nothing! ◡̈